pkgs/development/python-modules/openai/default.nix

```diff
@@ -68,10 +76,11 @@ buildPythonPackage rec {
   nativeCheckInputs = [
     inline-snapshot
     pytestCheckHook
+    dirty-equals
+    inline-snapshot
```

Comment:
`inline-snapshot` was already in there.

Reply:
Thanks, good catch, missed that during a merge.